src/AspNetCoreModuleV2/AspNetCore/PollingAppOfflineApplication.h

```diff
@@ -15,7 +15,7 @@ class PollingAppOfflineApplication: public APPLICATION
 {
 public:
     PollingAppOfflineApplication(IHttpApplication& pApplication, PollingAppOfflineApplicationMode mode)
-        :
+        : APPLICATION(pApplication),
```

Comment:
How is this related?

Reply:
Constructor argument was added to `APPLICATION`